app/scripts/directives/blocks.js

```diff
@@ -116,14 +116,29 @@ angular
           $scope.otherAnswer = '';
         }
 
-        $scope.$watchGroup(['selectedAnswer', 'otherAnswer'], () => {
-          if ($scope.answer) {
-            $scope.answer.value =
-              $scope.selectedAnswer === $scope.otherSentinel
-                ? $scope.otherAnswer
-                : $scope.selectedAnswer;
+        // Manually implement debounce so that $timeout can be mocked in tests
```

Comment:
A bit nitpicky, but:
```suggestion
        // Manually implement debounce in a way that $timeout can be mocked in tests
```
or

```suggestion
        // Manually implement debounce because debounce does not play well with radio buttons
```

We aren't manually implementing because we want `$timeout` to be mocked, but because debounce is not working well for radio buttons.

Reply:
Good point. I initially meant that I was writing a custom debouncer instead of using `_.debounce` because I needed it to use `$timeout`, but that's definitely not clear.